Test/spec/simd/simd_load.wast

```diff
@@ -186,23 +186,23 @@
 
 ;; Test operation with empty argument
 
-(assert_invalid
+(assert_malformed
```

Comment:
This should be totally a mistake I made before, I can't remember why I test v128 const here. :(

Any way, I can update the checkpoint later.